codechecker_lib/context_base.py

```diff
@@ -60,6 +61,9 @@ def set_env(self, env_vars):
         if codechecker_workspace:
             self._codechecker_workspace = codechecker_workspace
 
+        with open(self.checkers_severity_map_file) as severity_file:
```

Comment:
I think some error handling if the file is missing or it fails to load the json would be nice here.